packages/schema/src/Schema.ts

```diff
@@ -4405,15 +4413,15 @@ type MissingSelfGeneric<Usage extends string, Params extends string = ""> =
  * @category classes
  * @since 1.0.0
  */
-export interface Class<I, A, C, Self, Inherited = Data.Case> extends Schema<I, Self> {
+export interface Class<I, A, C, Self, Inherited> extends Schema<never, I, Self> {
```

Comment:
This should likely be `Class<R, I, A, C, Self, Inherited> extends Schema<R, I, Self>`